React/Base/macOS/RCTPlatformDisplayLink.m

```diff
@@ -65,6 +65,7 @@ - (void)dealloc
   if (_displayLink != NULL) {
     CVDisplayLinkStop(_displayLink);
     CVDisplayLinkRelease(_displayLink);
+    _displayLink = NULL;
```

Comment:
This also needs to have our macOS fork tag so it doesn't get wiped away in the next merge. Just add a `// TODO: GH#858` to it